openapi/platform/platform.yaml

```diff
@@ -465,6 +465,48 @@ paths:
               properties:
                 data:
                   $ref: '#/components/schemas/users'
+            examples:
+              Example 1:
+                value:
+                  data:
+                    type: users
+                    attributes:
+                      displayName: Bill Smith
```

Comment:
@chavanrohit123 verify the fields exposed in the doc and use those fields alone in the request example. Fields like emailSet, createdAt, updatedAt, profileImage etc are not in the docs. If it is optional, remove it, else add it in doc as wel..